src/vm/jitinterface.cpp

```diff
@@ -10882,6 +10882,33 @@ void CEEJitInfo::CompressDebugInfo()
     EE_TO_JIT_TRANSITION();
 }
 
+void increaseUnwindInfoSize(ULONG &unwindSize)
```

Comment:
This function adds space for the personality routine only, so it would be better to name it to reflect that.